tests/data/database_test.py

```diff
@@ -0,0 +1,9 @@
+import os
+
+from laa_crime_application_store_app.data.database import get_db
+
+
+def test_get_db_returns_valid_session():
+    os.environ["POSTGRES_NAME"] = "laa_crime_application_store_test"
```

Comment:
it feels like we have multiple approaches to the name setting.

Here we are hardcoding it, but in the migrations we appended to the existing name, I feel this is slightly confusing and would prefer a consistent approach if possible. :)

Reply:
Have normalised how we do this